pkg/matcher/README.md

```diff
@@ -21,19 +27,30 @@ Short Syntax
 
  Long Syntax
 
-     <line>      ::= [ <not> ] <format> <separator> <expr>
+     [ <not> ] <format> <separator> <expr>
      <format>    ::= [ 'string' | 'glob' | 'regexp' | 'simple_patterns' ]
      <not>       ::= '!'
                        negative expression
      <separator> ::= ':'
      <expr>      ::= any string
 ```
 
+**Tip**: In short syntax, you can enable the glob format by starting the string with a `*`, while in the long syntax you need to define it more explicitly. The following examples are identical. `simple_patterns` can be used **only** with the long syntax.
+
+Short Syntax: `'* * '`
+Long Syntax: `'glob':'*'`
+
 ### String matcher
 The string matcher reports whether the given value equals to the string ( use == ).
 
+e.g <br>
+`'== foo'` will match only if the string is `'foo'`. <br>
+`'[!] == bar'` will match any string that is not `'bar'`.
```

Comment:
i think the following would be better

```
Examples:
-   <EXAMPLE_1>
-   <EXAMPLE_2>
```
---

`[ ]` means **optional** (see any `man` page), so it is `!= PATTERN`